src/apis/iprocess_model_management_api.ts

```diff
@@ -196,4 +196,19 @@ export interface IProcessModelManagementApi {
     callback: Messages.CallbackTypes.OnProcessEndedCallback,
     subscribeOnce?: boolean,
   ): Promise<Subscription>;
+
+  /**
+   * Terminates a process.
+   *
+   * @async
+   * @param identity The requesting users identity.
```

Comment:
Please keep some alignment here:

```suggestion
   * @param identity          The requesting users identity.
```